socorro/cron/jobs/first_signatures.py

```diff
@@ -0,0 +1,204 @@
+import datetime
+import sys
+
+import mock
+
+from configman import Namespace
+from configman.converters import class_converter
+from crontabber.base import BaseCronApp
+from crontabber.mixins import (
+    as_backfill_cron_app,
+    with_postgres_transactions
+)
+
+from socorro.external.postgresql.signature_first_date import SignatureFirstDate
+from socorro.external.es.base import ElasticsearchConfig
+from socorro.external.es.supersearch import SuperSearch
+from socorro.external.es.super_search_fields import SuperSearchFields
+from socorro.app.socorro_app import App, main
+from socorro.lib.datetimeutil import string_to_datetime
+
+
+class SuperSearchErrors(Exception):
+    """Happens when we make a SuperSearch query and there's something in the
+    results 'errors' dict."""
+
+
+@with_postgres_transactions()
+@as_backfill_cron_app
+class FirstSignaturesCronApp(BaseCronApp):
+    app_name = 'first-signatures'
+    app_description = 'Using SuperSearch to log when signatures first appeared'
+    app_version = '0.1'
+
+    required_config = Namespace()
+    required_config.add_option(
+        'window_seconds',
+        default=60,  # seconds
```

Comment:
Why is this in seconds and not in minutes? Or at least, why is not equal to an hour?

Reply:
Yeah, it should be 60 * 60 by default.

Reply:
Fixed.